lib/rmt/mirror.rb

```diff
@@ -1,9 +1,10 @@
-module RMT
-end
-
 class RMT::Mirror
 
-  def initialize(mirroring_base_dir, repository_url, local_path, mirror_src = false, logger = nil)
+  class RMT::Mirror::Exception < RuntimeError
```

Comment:
For these empty classes, I personally prefer `class RMT::Mirror::Exception < RuntimeError; end`
I'm quite sure there is a RuboCop rule for that as-well. But I'm not insisting on that. It should just be same ruleset as in Glue :policeman: